vyper/venom/ir_node_to_venom.py

```diff
@@ -75,7 +76,6 @@
         "extcodehash",
         "balance",
         "msize",
-        "basefee",
```

Comment:
why is this removed?

Reply:
I think this is a duplicate since there is already a "basefee" at L50.